client/src/ui/NavBar.tsx

```diff
@@ -1,44 +1,53 @@
-import { Typography } from "@mui/material";
 import Link from "next/link";
+import Image from "next/image";
 
-interface ILinks {
```

Comment:
I should have been more explicit for this. When I said to place the new UI for the NavBar, I meant to say to combine what I have setup with the new changes, not remove what I wrote. It's easier to read and manage with my setup.